src/cmd/internal.go

```diff
@@ -113,6 +114,16 @@ var uiCmd = &cobra.Command{
 	},
 }
 
+var isValidHostname = &cobra.Command{
+	Use:   "is-valid-hostname",
+	Short: "Checks if the machine hostname is RFC1123 compliant",
```

Comment:
Per https://github.com/defenseunicorns/zarf/issues/1022 can we make it more clean that this is for the current machine?

```suggestion
	Short: "Checks if the current machine's hostname is RFC1123 compliant",
```